api/controllers/zetter.js

```diff
@@ -18,6 +18,21 @@ const createTweet = (req, res) => {
     res.status(201).json(newTweet);
 };
 
+const getProfile = (req, res) => {
+    const user = users.find((user) => user.username === req.user.username);
+
+    res.json(user);
+};
+
+const updateProfile = (req, res) => {
+    const user = users.find((user) => user.username === req.user.username);
+    user.username = req.body.username;
+    user.introduction = req.body.introduction;
+    user.email = req.body.email;
+    // もっとスマートにかけそう
```

Comment:
ここ、例えば
```
const userName = req.body.userName;
const introduction = req.body.introduction;

const {userName,  introduction} = newUser;
users.splice(user);
users.push(newUser);
```
みたいな方がスマート？
でも、結局
```
const userName = req.body.userName;
const introduction = req.body.introduction;
```
を書いてるから、まだ今の方法の方がマシ？

Reply:
うーん、今の書き方が一番読みやすいかなー

別な書き方だと
```suggestion
    index = users.findIndex(user => user.username === req.user.username)
    
    users[index] = {
        ...users[index],
        username: req.body.username,
        introduction: req.body.introduction,
        email: req.body.email,
    }
```

みたいな書き方はできるけど、あんまり読みやすくはないかな〜